src/Runtime/SymbolResolver.cs

```diff
@@ -72,9 +97,9 @@ public static IntPtr LoadImage (ref string name)
             return IntPtr.Zero;
         }
 
-        public static IntPtr ResolveSymbol (string name, string symbol)
+        public static IntPtr ResolveSymbol (string name, string symbol, string searchdir = null)
```

Comment:
Why do we need this parameter given that this method is only called by generated code?